packages/styles/components/Field.module.scss

```diff
@@ -9,6 +9,10 @@
   color: var(--gray-4);
   font: var(--typography-caption-font);
   margin-bottom: var(--space-xs);
+
+  input {
```

Comment:
We should avoid nesting here and improve a bit the semantic, so just add a classes called `checkbox` and `radio` to `input` and remove it as children of `label` 😊

Reply:
actually, the reason for this approach is because the 
```css
.container {
  display: flex;
  flex-flow: column;
}
```
If label and input stay in the same level, they will be arranged in a column.
if I change it inside the component, we won't have this style for the Vue version.

I would have to create a new .container-something or refactor the container to not have the `flex-flow: column;`